src/pages/settings/Wallet/ReportFraudPage.js

```diff
@@ -50,48 +51,40 @@ function ReportFraudPage({
 
     const domainCards = CardUtils.getDomainCards(cardList)[domain];
     const virtualCard = _.find(domainCards, (card) => card.isVirtual) || {};
+    const virtualCardError = ErrorUtils.getLatestErrorMessage(virtualCard) || '';
 
     const prevIsLoading = usePrevious(formData.isLoading);
 
+    // eslint-disable-next-line rulesdir/prefer-early-return
     useEffect(() => {
         if (prevIsLoading && formData.isLoading === false && _.isEmpty(virtualCard.errors)) {
             Navigation.navigate(ROUTES.SETTINGS_WALLET_DOMAINCARDS.getRoute(domain));
         }
-
-        if (formData.isLoading !== false && _.isEmpty(virtualCard.errors)) {
-            return;
-        }
-
-        FormActions.setErrors(ONYXKEYS.FORMS.REPORT_FRAUD_FORM, virtualCard.errors);
-    }, [domain, formData.isLoading, prevIsLoading, virtualCard.errors]);
-
-    const submitReportFraud = () => {
-        Card.reportDigitalExpensifyCardFraud(virtualCard.cardID);
-    };
-
-    const onBackButtonPress = () => Navigation.goBack(ROUTES.SETTINGS_WALLET_DOMAINCARDS.getRoute(domain));
+    }, [domain, formData, prevIsLoading, virtualCard.errors]);
```

Comment:
It's usually better to fix the lint errors than ignoring them.

```js
    useEffect(() => {
        if (!prevIsLoading || formData.isLoading) {
            return;
        }
        if (!_.isEmpty(virtualCard.errors)) {
            return;
        }

        Navigation.navigate(ROUTES.SETTINGS_WALLET_DOMAINCARDS.getRoute(domain));
    }, [domain, formData.isLoading, prevIsLoading, virtualCard.errors]);

```

Reply:
Right, updated